corehq/apps/userreports/models.py

```diff
@@ -495,6 +495,7 @@ class StaticDataSourceConfiguration(JsonObject):
     """
     _datasource_id_prefix = STATIC_PREFIX
     domains = ListProperty()
+    server_environment = ListProperty()
```

Comment:
will make this change once the rest of hte ucrs get filled out with environments